DESCRIPTION

```diff
@@ -31,6 +31,7 @@ Imports:
     lifecycle (>= 0.2.0),
     R6,
     rmarkdown (>= 2.19),
+    rtables (>= 0.5.1),
```

Comment:
@pawelru @shajoezhu  what do you think about moving it into imports? The library was anyway called directly in `to_flextable` (see this PR) and it was not checked (`if_installed`) beforehand, as far as I can see.

Reply:
I think it's ok. I dislike hiding dependencies in Suggests. This has a negative impact on `renv` and its dependency detection. If we cannot continue on missing package - this should go to imports.